src/util/utils.ts

```diff
@@ -96,6 +97,38 @@ export function stringTrimStart(str: string): [string, string] {
     return [ws, trimmed];
 }
 
+/**
+ * Checks a path is equal or a subpath of the other rootPath
+ *
+ * @param toCheck The path to check it is equal or a subpath of path.
+ * @param rootPath The ref path to check the other is equal to or a subpath of this.
+ * @tutorial
+ * rootPath = "root/sub/sub2"
+ * if toCheck = "notRoot/..." -> false
+ * if toCheck = "root" -> true
```

Comment:
I think I understand the confusion now. Yes we do have the same understanding of the word "sub", this might just be a documentation mistake?

`if toCheck = "root"` -> `true`
`if toCheck = "root/sub"` -> `true`

Should those be false? I think one of the test cases checks that it is false
`expect(isEqualOrSubPath(root, rootPath)).toBe(false);`

Reply:
BTW your English is great!

Reply:
Yes I think your right the documentation seems to be wrong. And now I understand what you mean with root path is not the correct naming. With ``rootPath``  I meant the path and all of its sub paths but It is not actually the root path. 

But the test should be correct if the path to check is above the ``rootPath`` it is not a part of the sub folders.